bitnami/jupyterhub/templates/hub/deployment.yaml

```diff
@@ -233,7 +233,7 @@ spec:
             - mountPath: /etc/jupyterhub/z2jh.py
               subPath: z2jh.py
               name: config
-            - mountPath: /etc/jupyterhub/secret/
+            - mountPath: /usr/local/etc/jupyterhub/secret/
```

Comment:
Do I need to mention this change in the README? I understand I do not need to bc the deployment will handle this once you upgrade the chart but just in case.

Reply:
This should not be a breaking change, so I would say we don't need to mention it.